Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -462,6 +464,19 @@
 #endif
 }
 
+bool WebPage::isTransparentOrFullyClipped(const Element& element) const
```

Comment:
This is a weird place for this function to live. WebPage has no business in asking questions about rendering.